crates/sui-config/src/node.rs

```diff
@@ -152,9 +152,10 @@ impl NodeConfig {
     }
 
     pub fn network_key_pair(&self) -> &NetworkKeyPair {
-        match self.network_key_pair.keypair() {
+        let sui_keypair = self.network_key_pair.keypair();
+        match sui_keypair {
             SuiKeyPair::Ed25519(kp) => kp,
-            _ => panic!("Invalid keypair type"),
+            _ => panic!("Invalid keypair type: {:?}", sui_keypair),
```

Comment:
Should you add the same panic info to `fn worker_key_pair(`?

Reply:
probably!